waspc/stack.yaml

```diff
@@ -18,7 +18,7 @@
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver: lts-13.7
+resolver: lts-16.14
```

Comment:
I updated it in order to get newer version of path library, because older version was not exposing Path.Posix directly.